src/Sentry.Android.AssemblyReader/Sentry.Android.AssemblyReader.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+
+    <!-- This package revs independently.  Bump this version manually when making changes. -->
+    <Version>1.0.0</Version>
+    <Description>.NET assembly reader for Android</Description>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Reflection.Metadata" Version="5.0.0" />
```

Comment:
It's what we use at the moment, yes.  It's a minimum version though, so it can be bumped higher if needed.
https://github.com/getsentry/sentry-dotnet/blob/d985e7b56fb8fc567a26a977f592562835c95ee3/src/Sentry/Sentry.csproj#L33-L35

We've already got this in the dependency chain for Xamarin.